### PR TITLE
Aggies Sprint 9

### PR DESCRIPTION
**Pull request by rhundal7**
[UIWRKFLOW-33](https://github.com/folio-org/ui-workflow/pull/68)
[UIWRKFLOW-34](https://github.com/folio-org/ui-workflow/pull/66)
[UIWRKFLOW-35](https://github.com/folio-org/ui-workflow/pull/69)

### CODE REVIEW

File: src/components/icon/GraphItemIcon/GraphItemIcon.tsx [hunk 1]
```diff
@@ -86,7 +86,7 @@ export const GraphItemIcon: React.FC<{ type: string }> = ({ type }) => {
   }
 
   return <div className={css?.graphItemtIconTypeDiv}>
-    <img alt="Graph Item Icon" src={icon} width="22" height="22" />
+    <AppIcon app="@folio/workflow" iconKey="app" size="small" src={icon} />
```
Comment (kaladay) [inline src/components/icon/GraphItemIcon/GraphItemIcon.tsx:89]: I went ahead and created a Tech Debt issue to address the introduction of the use of `AppIcon` here.
- https://folio-org.atlassian.net/browse/UIWRKFLOW-38

Comment (wwelling) [reply src/components/icon/GraphItemIcon/GraphItemIcon.tsx:89]: Not sure why there are two icon components. Is is that AppIcon is only expected to work in the context of the header navbar and Icon should be used elsewhere? The AppIcon does work as expected in this context but is preferred to follow which the framework prescribes.

Thanks for the follow up issue.

Comment (wwelling) [reply src/components/icon/GraphItemIcon/GraphItemIcon.tsx:89]: Here is another reference use of `AppIcon`. Again, still not sure what this component is for or how it differs from the `Icon` component.

https://github.com/folio-org/stripes-components/blob/b0ec27770e4a32c50d4e0af8e871391158535035/lib/PaneHeader/readme.md

Perhaps we should create a WorkflowNodeIcon similar to CustomIcon here https://github.com/folio-org/stripes-components/blob/b0ec27770e4a32c50d4e0af8e871391158535035/lib/Icon/stories/CustomIcon.js#L4?

File: src/components/icon/GraphItemIcon/GraphItemIcon.module.css [hunk 2]
```diff
@@ -4,5 +4,6 @@
 }
 
 .graphItemtIconTypeDiv span {
```
Comment (wwelling) [inline src/components/icon/GraphItemIcon/GraphItemIcon.module.css:6]: Styling the AppIcon or any inner Icon component via a wrapper class and child element `span` is potential for regression. If AppIcon or Icon changes the HTML element from `span` some other markup tag, the CSS rules will no longer apply.

Comment (wwelling) [reply src/components/icon/GraphItemIcon/GraphItemIcon.module.css:6]: It does appear these CSS rules will still apply to `Icon` component but uncertain of its computed styles.

https://github.com/folio-org/stripes-components/blob/master/lib/Icon/Icon.js#L96

Comment (wwelling) [reply src/components/icon/GraphItemIcon/GraphItemIcon.module.css:6]: And here is the stripes-core `AppIcon` component documentation. https://github.com/folio-org/stripes-core/tree/master/src/components/AppIcon

File: src/components/icon/GraphItemIcon/GraphItemIcon.tsx [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-
+import { AppIcon } from '@folio/stripes/core';
 import {
```
Comment (kaladay) [inline src/components/icon/GraphItemIcon/GraphItemIcon.tsx:3]:
```suggestion
import { AppIcon } from '@folio/stripes/core';

import {
```

The style practice we are following (and need to document) is to separate the types of imports by new lines.

The local files should have a new line.

**Review by kaladay — CHANGES_REQUESTED** [review]

File: src/components/common/ItemValues/GraphItemValue.module.css [hunk 4]
```diff
@@ -1,17 +1,16 @@
 @import "@folio/stripes-components/lib/variables";
 
+.nodeCard:hover,
```
Comment (kaladay) [inline src/components/common/ItemValues/GraphItemValue.module.css:3]: I would note that for [UIWRKFLOW-35](https://folio-org.atlassian.net/browse/UIWRKFLOW-35):
> The selected state styles should not change on hover. Selection should override hover.

This implementation doesn't overide.
It uses the same state for both.

Comment (wwelling) [reply src/components/common/ItemValues/GraphItemValue.module.css:3]: Noted.

File: src/components/icon/GraphItemIcon/GraphItemIcon.tsx (hunk 1 above)
Comment (wwelling) [reply src/components/icon/GraphItemIcon/GraphItemIcon.tsx:90]: It has been decided `AppIcon` is the correct component to use here.

**Review by kaladay — APPROVED** [review]

**Pull request merged**